src/main/java/sixgaezzang/sidepeek/projects/util/validation/FileValidator.java

```diff
@@ -1,15 +1,22 @@
 package sixgaezzang.sidepeek.projects.util.validation;
 
+import static sixgaezzang.sidepeek.projects.exception.message.FileErrorMessage.OVERVIEW_IMAGE_OVER_MAX_COUNT;
+import static sixgaezzang.sidepeek.projects.exception.message.FileErrorMessage.OVERVIEW_IMAGE_URL_IS_INVALID;
+import static sixgaezzang.sidepeek.projects.exception.message.FileErrorMessage.OVERVIEW_IMAGE_URL_OVER_MAX_LENGTH;
 import static sixgaezzang.sidepeek.projects.util.ProjectConstant.MAX_OVERVIEW_IMAGE_COUNT;
 
 import io.jsonwebtoken.lang.Assert;
 import java.util.List;
+import sixgaezzang.sidepeek.common.util.ValidationUtils;
 
 public class FileValidator {
```

Comment:
```suggestion
public class FileValidator {

    private FileValidator() {}
```

`Validator` 클래스들도 상속이나 확장할 경우가 없다면, private 생성자를 사용해 객체 생성을 제한할 수 있으면 좋을 것 같습니다!